textattack/constraints/grammaticality/language_models/google_language_model/google_language_model.py

```diff
@@ -75,6 +78,9 @@ def get_probs(current_text, transformed_texts):
             probs_of_swaps_at_index = list(
                 zip(item_indices, get_probs(current_text, this_transformed_texts))
             )
+            # if len(probs_of_swaps_at_index) == 0:
+            #     probs.extend(0)
+            #     continue
```

Comment:
if we don't need this block here, maybe we can remove it?

Reply:
thanks! will delete